test/e2e/service_test.go

```diff
@@ -133,3 +139,19 @@ func serviceMultipleDelete(r *test.KnRunResultCollector, existService, nonexistS
 	assert.Check(r.T(), strings.Contains(out.Stdout, expectedSuccess), "Failed to get 'successfully deleted' message")
 	assert.Check(r.T(), strings.Contains(out.Stdout, expectedErr), "Failed to get 'not found' error")
 }
+
+func serviceDeleteAll(r *test.KnRunResultCollector, service1 string, service2 string, service3 string) {
+	out := r.KnTest().Kn().Run("service", "list")
+	r.AssertNoError(out)
+	assert.Check(r.T(), strings.Contains(out.Stdout, service1), "The service ", service1, " does not exist (but is expected to exist)")
+	assert.Check(r.T(), strings.Contains(out.Stdout, service2), "The service ", service2, " does not exist (but is expected to exist)")
+	assert.Check(r.T(), strings.Contains(out.Stdout, service3), "The service ", service3, " does not exist (but is expected to exist)")
+
+	out = r.KnTest().Kn().Run("service", "delete", "--all")
+	r.AssertNoError(out)
+
+	namespace := r.KnTest().Kn().Namespace()
+	expectedSuccess := fmt.Sprintf("Service '%s' successfully deleted in namespace '%s'.\nService '%s' successfully deleted in namespace '%s'.\nService '%s' successfully deleted in namespace '%s'.\n",
+		service3, namespace, service2, namespace, service1, namespace)
+	assert.Check(r.T(), strings.Contains(out.Stdout, expectedSuccess), "Failed to get 'successfully deleted' message")
```

Comment:
```suggestion
	assert.Check(r.T(), util.ContainsAll(out.Stdout,  "deleted", service1, service2, service3, namespace), "failed to get expected strings in message")
```

Reply:
instead of checking the exact message, we can also check for certain substrings in the message, other tests are verifying in the same manner, not a blocker though.